README.md

```diff
@@ -1,6 +1,6 @@
 # pytomlpp
 
-[![Build Status](https://travis-ci.com/bobfang1992/pytomlpp.svg?branch=master)](https://travis-ci.com/bobfang1992/pytomlpp)
+[![Build Status](https://github.com/bobfang1992/pytomlpp/workflows/Wheels/badge.svg)](https://travis-ci.com/bobfang1992/pytomlpp)
```

Comment:
Could we also change the fallback url from Travis to Github Action? Thanks.

Reply:
Not actually a fallback URL, rather the link the user is taken to when they click on it